test/CMakeLists.txt

```diff
@@ -47,7 +64,7 @@ if (WITH_TEST_GENERATORS)
   # and make an executable target for it named 'generator_foo'
   file(GLOB TESTS RELATIVE "${CMAKE_CURRENT_SOURCE_DIR}/generator" "${CMAKE_CURRENT_SOURCE_DIR}/generator/*_jittest.cpp")
   foreach(test ${TESTS})
-    string(REPLACE "_jittest.cpp" "" name "generator_jit_${test}")
+    string(REPLACE "_jittest.cpp" "" name "generator_exec_jit_test_${test}")
```

Comment:
This is a dumb problem, but...

The windows buildbots currently have problems with really long file names (>260 characters). Indeed, I'm currently running a build slave in a directory called E:/b/ to avoid this. The problem seems to be that generators end up in deeply nested folders that contain the name of the generator twice. Perhaps this change could be modified to not extend the length of any filenames?